### PR TITLE
Add roundToNearest(n:) extension to Int

### PR DESCRIPTION
**Pull request by Vyax**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 4.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a changelog entry describing my changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - Added `queryParmeters` property to get the query parameters from a URL as a dictionary. [#370](https://github.com/SwifterSwift/SwifterSwift/pull/370) by [nathanbacon](https://github.com/nathanbacon).
 - **Array**
   - added `divided(by:)` to separate an array into 2 arrays based on a predicate. [#367](https://github.com/SwifterSwift/SwifterSwift/pull/367) by [@neoneye](https://github.com/neoneye).
+- **Int**
+  - added `roundToNearest(n:)` to round an integer to the closest multple of a given n. [#381](https://github.com/SwifterSwift/SwifterSwift/pull/381) by [@vyax](https://github.com/vyax).
```
Comment (LucianoPAlmeida) [inline CHANGELOG.md]: Typo here on "multple"? Do you mean multiple?

**Comment by Vyax** [comment]
Okay, I fixed the typo.

File: Sources/Extensions/SwiftStdlib/IntExtensions.swift [hunk 2]
```diff
@@ -137,6 +137,14 @@ public extension Int {
 		}
 		return romanValue
 	}
+    
+    /// SwifterSwift: Rounds to the closest multiple of n
+    ///
+    ///        12.roundToNearest(5) -> 10
+    ///        59.roundToNearest(25) -> 50
```
Comment (SD10) [inline Sources/Extensions/SwiftStdlib/IntExtensions.swift]: I'm not going to lie, code examples are really starting to bother me 😅 I'd much prefer a comment explaining that we use traditional grade school rounding and round up after 5

Comment (Vyax) [reply Sources/Extensions/SwiftStdlib/IntExtensions.swift]: Alright, I removed the examples

**Comment by Vyax** [comment]
@omaralbeik Why did you merge master into this branch? I could just rebase onto SwifterSwift/master.

**Comment by SD10** [comment]
We squash all PRs on merge so it doesn't really matter. We need to formalize our workflow. See #377. I'm going to do that this weekend

**Comment by omaralbeik** [comment]
@Vyax I tried to solve a conflict in the changelog, sorry I should have asked before 😅

**Pull request merged**